src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/Utils.scala

```diff
@@ -31,6 +31,8 @@ import scala.jdk.CollectionConverters._
 import scala.util.Try
 object Utils {
 
+  val PAGOPA = "PagoPA"
```

Comment:
This is the name currently stored for the tenant
```suggestion
  val PAGOPA = "PagoPA S.p.A."
```